### DISCUSSION TITLE
Throttle ramp up

### DISCUSSION THREAD

**Discussion by Energeek136** [danielnilsson9/bbs-fw#228] [General]
Is there any update about limiting too fast throttle ramp up?

After going through three different firmwares, setting start current, etc the only slightly better option is to limit the throttle current at each pas level and then the startup would be alot smoother but then I am stuck that my throttle is limited to that current. 

After driving tsdz8 their throttle ramp up is super smooth and slow which I love.

*I am running a bbs02b, if someone managed to limit to throttle on bbshd let me know please

**Comment by dav0000000** [comment]
Have you tried making the ramp up slower in the config?

**Reply by Energeek136** [reply]
Hey, thanks for answering. Which config specificaly? Do you mean inside the bbs-fw tool?
If yes then I already tried to decrease the Current Ramp(A/s) and putting any number does not change anything.

Also in the tool start current(%) for throttle is at 1%

**Reply by dav0000000** [reply]
Yes that is what I was talking about.

**Reply by Energeek136** [reply]
Unfortunately none of those do any change for me, only limiting the current of throttle for each pas setting does

**Comment by Energeek136** [comment]
Is it possible to hard code the ramp up inside the build?

**Reply by Paulii6661** [reply]
I did you can check the discussion here for quadratic throttle I wrote code there you can just copy a and paste it and compile after that you don't need to change pass because even in 100% power it's very smooth
